<?xml version="1.0"?>
		
<launch>

	<!-- <group ns="robot"> -->

		<!-- Arguments for led driver -->
		<arg name = "robot_model"     default = "debug"/>
		<arg name = "leds_port"		 default = "/dev/ttyUSB_LEDS"/>
		<arg name = "exclusive_signals" default= "false"/>

		<!-- Arguments for user application example -->
		<arg name="cmd_vel"			default = "robotnik_base_control/cmd_vel"/>

		<!-- Launch user application example-->
		<node pkg="robotnik_leds_example" type="leds_robot_debug.py" name="leds_robot_example_node" output="screen">
			<param name="cmd_vel" value="$(arg cmd_vel)"/>
		</node>

		<!-- Launch led driver -->
		<include file="$(find robotnik_signal_leds_manager)/launch/leds_manager.launch">
			<arg name="robot_model" value="$(arg robot_model)"/>
			<arg name="leds_port" value="$(arg leds_port)"/>
			<arg name="exclusive_signals" value = "$(arg exclusive_signals)"/>
		</include>

		
	<!-- </group> -->

</launch>